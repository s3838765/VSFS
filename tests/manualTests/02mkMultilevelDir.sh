#! /usr/bin/sh
cp 02mkMultilevelDir.notes 02mkMultilevelDirTemp.notes
../../VSFS mkdir 02mkMultilevelDirTemp.notes dir1/dir2/dir3/dir4/
cp 02mkMultilevelDirTemp.notes 02mkMultiLevelDir.out
rm 02mkMultilevelDirTemp.notes
