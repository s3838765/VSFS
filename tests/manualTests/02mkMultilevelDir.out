NOTES V1.0
=dir1/
=dir1/dir2/
=dir1/dir2/dir3/
=dir1/dir2/dir3/dir4/
